test-configs.yaml

```diff
@@ -1773,7 +1778,6 @@ test_configs:
 
   - device_type: qemu_arm-versatilepb
     test_plans:
-      - baseline_qemu
```

Comment:
Why remove baseline?  The boot tests will soon disappear and this platform won't be tested at all any more.

Reply:
I restore baseline. I have sent a defconfig change that should be merged soon.